week6/credit.py

```diff
@@ -0,0 +1,38 @@
+def main():
+    card_number = input("Enter card number: ")
+    length = len(card_number)
+    x = 0
```

Comment:
```suggestion
    card_number_length = len(card_number)
    x = 0
```
Was ist x für eine Variable? 😁